src/util/commit-split.ts

```diff
@@ -114,8 +114,15 @@ export class CommitSplit {
         splitCommits[pkgName].push(commit);
       }
       if (commit.files.length === 0 && this.includeEmpty) {
-        for (const pkgName in splitCommits) {
-          splitCommits[pkgName].push(commit);
+        if (this.packagePaths) {
```

Comment:
My understanding is that `this.packagePaths` will be populated with all the package names, when you're using a manifest?

It might be worth adding a comment that mentions what this defaults to, as I had to read through the codebase to figure this out.

Reply:
Done, added comments on the options.